packages/flutter/lib/src/rendering/list_wheel_viewport.dart

```diff
@@ -15,8 +15,37 @@ import 'viewport_offset.dart';
 
 typedef double _ChildSizingFunction(RenderBox child);
 
+/// A delegate used by [RenderListWheelViewport] to manage its children.
+///
+/// [RenderListWheelViewport] during layout will ask the delegate to create
+/// children that are visible in the viewport and remove those that are not.
+abstract class ListWheelChildManager {
+  /// The number of children that will be provided.
+  ///
+  /// If non-null, the children will have index in the range [0, childCount - 1].
+  ///
+  /// If null, then there's no explicit limits to the range of the children
+  /// except that it has to be contiguous. If [childExistsAt] for a certain
+  /// index returns false, that means the index is either a lower or upper limit.
```

Comment:
Be exact with the wording. If index x returns false, x is already past the limit.

Reply:
fixed.